plugins/processors/awsappsignals/internal/resolver/attributesresolver.go

```diff
@@ -69,34 +92,67 @@ func (r *attributesResolver) Stop(ctx context.Context) error {
 	return errs
 }
 
-type hostedInAttributeResolver struct {
-	name         string
+type resourceAttributesResolver struct {
+	platformCode string
+	platformType string
 	attributeMap map[string]string
 }
 
-func newHostedInAttributeResolver(name string, attributeMap map[string]string) *hostedInAttributeResolver {
-	if name == "" {
-		name = AttributePlatformGeneric
-	}
-	return &hostedInAttributeResolver{
-		name:         name,
+func newResourceAttributesResolver(platformCode, platformType string, attributeMap map[string]string) *resourceAttributesResolver {
+	return &resourceAttributesResolver{
+		platformCode: platformCode,
```

Comment:
whats the difference between `platformCode` and `platformType`?

Reply:
Renamed `platformCode` to `defaultEnvPrefix`.